shared/src/vyxal/Interpreter.scala

```diff
@@ -8,6 +8,7 @@ import scala.collection.mutable.ListBuffer
 import scala.collection.mutable as mut
 
 object Interpreter:
+  def version = "3.0.0"
```

Comment:
At some point, we may be able to use https://github.com/sbt/sbt-release instead of hard-coding the version number. That's for later, though, let's not delay v3 further